index.html

```diff
@@ -4372,6 +4372,146 @@ <h2>Revision History</h2>
+all the proofs in a <em>proof set</em> on a document that has been signed by the
+VP of engineering, CFO, and CEO:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will verify
+  &quot;proof&quot;: [
+  {verificationMethod: pubVPE, proofValue: signature(privVPE, document)},
+  {verificationMethod: pubCFO, proofValue: signature(privCFO, document)},
+  {verificationMethod: pubCEO, proofValue: signature(privCEO, document)}]
+}
+      </code></pre>
+      <p>
+The <em>holder</em> or any other intermediary receiving a signed VC containing a
+<em>proof set</em> is able to remove any of the <em>proofs</em> within the set
+prior to passing it on to another entity and this VC will still verify. This may
+or may not be the intent. For the signatories sending a birthday card to a
+valued employee this is probably fine. If we are trying to model a business
+process where approvals ascend the company hierarchy this would not be ideal
+since any intermediary could remove signatures from the <em>proof set</em> and
+still have it verify, i.e., below it looks like the CFO and CEO just approved
+something without the VP of engineering's concurrence.
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will also verify
+  &quot;proof&quot;: [
+  {verificationMethod: pubCFO, proofValue: signature(privCFO, document)},
+  {verificationMethod: pubCEO, proofValue: signature(privCEO, document)}]
+}
+      </code></pre>
+      <p>
+We can introduce a &quot;dependency relation&quot; between <em>proofs</em> in a
+<em>proof set</em> by assigning the proof that another proof relies on, i.e.,
+the <em>dependent proof</em>, an <em>id</em> and the <em>relying proof</em> will
+have a list (or single item) denoting <em>previousProofs</em>. Such
+<em>dependency chains</em> can have arbitrary depth. The <strong>intent</strong>
+of such a <em>proof chain</em> is to model an approval chain in a business
+process or a notary witnessing analog signatures. In such cases we CANNOT allow
+the <em>proofs</em> that are relied upon to be <strong>substituted</strong> with
+other proofs or removed.
+      </p>
+      <p>
+Let's see how a <em>proof chain</em> is constructed where the VP of Engineering
+signs off on the document first, based on the VP of engineering's signature and
+a review the CFO then signs off of the document, finally based on both the prior
+signatures and a review the CEO signs off on the document. Since others will be
+referring to the VP of engineering's signature we need to add a <em>id</em> to
+the proof options. First the VC of engineering signs the VC:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will verify
+  &quot;proof&quot;: {id: 0, verificationMethod: pubVPE, proofValue: signature(privVPE, document)}
```

Comment:
Does `id` have to be a URI here?

Reply:
Yes, it does ... I overlooked that and I get that perhaps it was meant to be illustrative, but we should definitely make it a URI to avoid confusion. Perhaps `urn:proof-0` or something like that can be done to help people follow the example.